<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(75, 75, 75);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Password Reset</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="otp_passreset_lineEdit">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom: 2px solid rgbargb(255, 255, 255, 210);
color:rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
              </property>
              <property name="placeholderText">
               <string>Enter OTP</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="new_password_lineEdit">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom: 2px solid rgbargb(255, 255, 255, 210);
color:rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Enter New Password</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="confirm_new_lineEdit_2">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom: 2px solid rgbargb(255, 255, 255, 210);
color:rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Re-Enter New Password</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="confirm_pushButton">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#confirm_pushButton{
	
	background-color: rgb(0, 0, 0);
	color:rgba(255, 255, 255, 210);
	border-radius: 5px;
}

QPushButton#confirm_pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(139, 139, 139));
}

QPushButton#confirm_pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgba(139, 139, 139);
}</string>
              </property>
              <property name="text">
               <string>Confirm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="goToGenerate_pushButton">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#goToGenerate_pushButton{
	
	background-color: rgb(0, 0, 0);
	color:rgba(255, 255, 255, 210);
	border-radius: 5px;
}

QPushButton#goToGenerate_pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(139, 139, 139));
}

QPushButton#goToGenerate_pushButton:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgba(139, 139, 139);
}</string>
              </property>
              <property name="text">
               <string>Go Back</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
